File: Level 064/homework/hw8.js
// 8) მომხარებელს შემოატანინე input შეამოწმე, არის თუ არა რიცხვი 10-ზე მეტი ან ტოლი.

let input = prompt("Enter num: ")

if (input < 10) {
    console.log("ნაკლებია")
}

else if (input > 10) {
    console.log("მეტია")
}

else {
    console.log("ტოლია")
}

File: Level 014/homework/hw8.py
# )შემოატანინეთ მომხმარებელს რიცხვი და დაადგინეთ არის თუ არა დადებითი უარყოფითი ან ნულის ტოლი

user_input = int(input("Enter num: "))

if user_input > 0:
    print("დადებითია")
elif user_input < 0:
    print("უარყოფითია")
else:
    print("ნულის ტოლია")